setup.sh

```diff
@@ -195,7 +195,8 @@ fi
 # Install dependencies from requirements.txt
 cd $run_name_folder_path && pip install --upgrade pip
 if [[ "$MODE" == "pinned" ]]; then
-    pip3 install -r requirements.txt -c constraints_gpu.txt
```

Comment:
I think adding `-U` here goes against the idea of pinned build. Previously, the versions installed will be the function of the base image (which is pinned) and reqs and constraints. Now it could potentially be time-dependent if we happen to miss to capture the constraints in the constraints file. WDYT? @khatwanimohit

Reply:
I understand, we can remove it.